src/jobs/qa-automation-suite.yaml

```diff
@@ -0,0 +1,48 @@
+parameters:
+  test_path:
+    description: "The path to the test suite to run"
+    type: string
+  project_name:
+    description: "The name of the project in report portal"
+    type: string
+  launch_name:
+    description: "The name to give to the test launch"
+    type: string
+    default: EXTERNAL_TEST_RUN
+  launch_description:
+    description: "The description to give to the test launch"
+    type: string
+    default: "Automated Tests Ran by CircleCI"
+  launch_tag:
+    description: "The tags to add to the test launch (seperated by semi-colons)"
+    type: string
+    default: CIRCLECI;EXTERNAL
+  suite_version:
+    description: "The qa-automation-tests version to use"
+    type: string
+    default: "latest"
+
+executor:
+  name: qa-automation
+  qa_automation_runner_version: << parameters.suite_version >>
+
+environment:
+  TEST_PATH: << parameters.test_path >>
+  PROJECT_NAME: << parameters.project_name >>
+  LAUNCH_NAME: << parameters.launch_name >>
+  LAUNCH_DESCRIPTION: << parameters.launch_description >>
+  LAUNCH_TAG: << parameters.launch_tag >>
+  RP_ENDPOINT: https://reportportal.dev.codacy.org
+  SELENIUM_DRIVER_URL: http://localhost:4444/wd/hub
+  HUB_URL: http://localhost:4444/wd/hub
+  JAVA_OPTS: -Xss256m -Xmx512m
+  MAVEN_OPTS: -showversion -Xms256m -Xmx512m
```

Comment:
this scares me a little, we are not recompiling all the times right?
also again if the configuration is always needed it should probably be in the docker image.

Reply:
This configs are the one used in the codacy/chart. I just copied them from there in order to move this to an orb.
I don't know why they are needed. @jonathancarvalheiro @medleycodacy any idea?

Reply:
@andreaTP we're not compiling, we are running directly the tests. This options usually are required to handle a great number of tests. I can do a PR adding this as default to the docker image.

Reply:
Created a PR for this: https://bitbucket.org/qamine/qa-automation-tests/pull-requests/219/feature-add-envs-mvn_opts-and-java_opts-to/diff

Reply:
approved there :+1: